test/main/deepEqual.js

```diff
@@ -1581,6 +1581,35 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 	}
 );
 
+QUnit.test( "Compare primitive objects", function( assert ) {
+	var SafeNumber = Number,
+	SafeString = String,
+	SafeBoolean = Boolean;
```

Comment:
@leobalter 
For lint validation. like it [lint](https://github.com/jquery/qunit/blob/master/test/main/deepEqual.js#L76)

:)